src/main/webapp/src/services/BackendService.js

```diff
@@ -23,3 +23,13 @@ export function deleteCourse(courseId) {
 export function getCategories() {
     return axios.get('categories');
 }
+
+export function createFeedback(courseId, feedback) {
+    const body = {
```

Comment:
because feedback in frontend is participantName and in backend is participant_name

Reply:
our naming convention in frontend is camelCase. do you think I should just rename this one to participant_name ? it will save some code but it won't be consistent